README.md

```diff
@@ -3,115 +3,94 @@
 ![Coverage badge](https://img.shields.io/endpoint.svg?url=https%3A%2F%2Fsilent-zebra-36.tunnel.datahub.at%2Fcoverage%2Frigbox%2Fmaster)
 ![Build status badge](https://img.shields.io/endpoint.svg?url=https%3A%2F%2Fsilent-zebra-36.tunnel.datahub.at%2Fstatus%2Frigbox%2Fmaster)
 
-Rigbox is a high-performance, open-source MATLAB toolbox for managing behavioral neuroscience experiments. Initially developed to probe mouse behavior for the [Steering Wheel Setup](https://www.ucl.ac.uk/cortexlab/tools/wheel),  Rigbox simplifies hardware/software interfacing and creates a runtime environment in which an experiment's parameters can be easily monitored and manipulated.
```

Comment:
the post-comma phrase doesn't really follow the pre-comma. The sentence should read something like "Rigbox was initially developed to do this, and now also does this". 

> Besides, we're on GitHub so users should be able to see that the repo is active.

I don't think you should assume someone who is on the landing page is immediately going to check the commit history dates.

Reply:
Saying that is "under active" development makes it sound like this is not a stable package. I think it's better to remove that. 
Also, test-driven development is a very specific thing (https://en.wikipedia.org/wiki/Test-driven_development). You don't need say that on the README at all. You can say that in the contributions -- so that a new pull request should abide by test-driven development. Moreover, maybe what you mean is that the software is carefully tested, not that it abdies by the "test-driven development" philosophy (i.e. add test for new feature, then write code until that test and all tests pass).